tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -112,6 +112,11 @@ private async void RenderWidgetFrameworkElement()
     {
         var cardTemplate = await Widget.GetCardTemplateAsync();
         var cardData = await Widget.GetCardDataAsync();
+        var hostData = new JsonObject

```

Comment:
Can you move this down to where we use the host data? That way if we error out in between, we don't create this object that will ultimately not be used.